### PR TITLE
chore: update to latest axios everywhere

### PR DESCRIPTION
**Pull request by gre**
### 📝 Description

- followed migration guide https://github.com/axios/axios/discussions/4996#discussioncomment-5581701
- refined some typing of live-network

### ❓ Context

- **JIRA or GitHub link**: na

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [x] **Impact of the changes:** networks

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: apps/ledger-live-desktop/package.json [hunk 1]
```diff
@@ -186,7 +186,7 @@
     "@types/write-file-atomic": "^4.0.0",
     "@vitejs/plugin-react": "^3.1.0",
     "allure-playwright": "2.12.2",
-    "axios": "^1.3.4",
+    "axios": "^1.6.7",
```
Comment (jdabbech-ledger) [inline apps/ledger-live-desktop/package.json:189]: This update broke a test on ci recently https://ledger.slack.com/archives/C03MHGMAMRQ/p1708699888332929?thread_ts=1708689487.052639&cid=C03MHGMAMRQ

Comment (gre) [reply apps/ledger-live-desktop/package.json:189]: checking 👀

**Comment by gre** [comment]
problematic libs that are still on 0.x of axios
- tronweb
- @taquito/*
- @cosmjs/*
- @elrondnetwork/*
- @keplr-wallet/*

```
pnpm desktop why axios


dependencies:
@ledgerhq/coin-evm link:../../libs/coin-evm
├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
│ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│   └── axios 1.6.7
├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│ └── axios 1.6.7
├─┬ @ledgerhq/domain-service link:../../libs/domain-service
│ └── axios 1.6.7
├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
│ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│ │ └── axios 1.6.7
│ └── axios 1.6.7
├─┬ @ledgerhq/live-network link:../../libs/live-network
│ └── axios 1.6.7
└── axios 1.6.7
@ledgerhq/coin-framework link:../../libs/coin-framework
└─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  └── axios 1.6.7
@ledgerhq/domain-service link:../../libs/domain-service
└── axios 1.6.7
@ledgerhq/hw-transport-http link:../../libs/ledgerjs/packages/hw-transport-http
└── axios 1.6.7

devDependencies:
@ledgerhq/test-utils link:../../libs/test-utils
└─┬ @ledgerhq/live-common link:../../libs/ledger-live-common
  ├─┬ @cosmjs/stargate 0.26.6
  │ └─┬ @cosmjs/tendermint-rpc 0.26.6
  │   └── axios 0.21.4
  ├─┬ @crypto-org-chain/chain-jslib 1.1.2
  │ ├─┬ @cosmjs/proto-signing 0.24.1
  │ │ └─┬ @cosmjs/launchpad 0.24.1
  │ │   └── axios 0.21.4
  │ ├─┬ @cosmjs/stargate 0.25.0-alpha.2
  │ │ └─┬ @cosmjs/tendermint-rpc 0.25.0-alpha.2
  │ │   └── axios 0.21.2
  │ ├─┬ @cosmjs/tendermint-rpc 0.25.0-alpha.2
  │ │ └── axios 0.21.2
  │ └── axios 0.21.2
  ├─┬ @elrondnetwork/erdjs-network-providers 1.2.0
  │ └── axios 0.24.0
  ├─┬ @keplr-wallet/cosmos 0.9.16
  │ ├─┬ @cosmjs/launchpad 0.24.1
  │ │ └── axios 0.21.4
  │ ├─┬ @keplr-wallet/types 0.9.12
  │ │ ├─┬ @cosmjs/launchpad 0.24.1
  │ │ │ └── axios 0.21.4
  │ │ ├─┬ @cosmjs/proto-signing 0.24.1
  │ │ │ └─┬ @cosmjs/launchpad 0.24.1
  │ │ │   └── axios 0.21.4
  │ │ ├── axios 0.21.4
  │ │ └─┬ secretjs 0.17.5
  │ │   └── axios 0.21.1
  │ ├─┬ @keplr-wallet/unit 0.9.12
  │ │ └─┬ @keplr-wallet/types 0.9.12
  │ │   ├─┬ @cosmjs/launchpad 0.24.1
  │ │   │ └── axios 0.21.4
  │ │   ├─┬ @cosmjs/proto-signing 0.24.1
  │ │   │ └─┬ @cosmjs/launchpad 0.24.1
  │ │   │   └── axios 0.21.4
  │ │   ├── axios 0.21.4
  │ │   └─┬ secretjs 0.17.5
  │ │     └── axios 0.21.1
  │ └── axios 0.21.4
  ├─┬ @ledgerhq/coin-algorand link:../../libs/coin-algorand
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-evm link:../../libs/coin-evm
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/domain-service link:../../libs/domain-service
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
  │ │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ │ └── axios 1.6.7
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/live-network link:../../libs/live-network
  │ │ └── axios 1.6.7
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-near link:../../libs/coin-near
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-polkadot link:../../libs/coin-polkadot
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/hw-app-eth link:../../libs/ledgerjs/packages/hw-app-eth
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/domain-service link:../../libs/domain-service
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
  │ │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ │ └── axios 1.6.7
  │ │ └── axios 1.6.7
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/hw-app-vet link:../../libs/ledgerjs/packages/hw-app-vet
  │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/hw-transport-node-speculos-http link:../../libs/ledgerjs/packages/hw-transport-node-speculos-http
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/live-countervalues link:../../libs/live-countervalues
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/live-countervalues-react link:../../libs/live-countervalues-react
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-countervalues link:../../libs/live-countervalues
  │   ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │   │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   │   └── axios 1.6.7
  │   ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   │ └── axios 1.6.7
  │   └─┬ @ledgerhq/live-network link:../../libs/live-network
  │     └── axios 1.6.7
  ├─┬ @ledgerhq/live-network link:../../libs/live-network
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/live-nft link:../../libs/live-nft
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @stellar/stellar-sdk 11.2.2
  │ └── axios 1.6.7
  ├─┬ @taquito/ledger-signer 13.0.1
  │ └─┬ @taquito/taquito 13.0.1
  │   ├─┬ @taquito/http-utils 13.0.1
  │   │ └── axios 0.26.1
  │   ├─┬ @taquito/michelson-encoder 13.0.1
  │   │ └─┬ @taquito/rpc 13.0.1
  │   │   └─┬ @taquito/http-utils 13.0.1
  │   │     └── axios 0.26.1
  │   └─┬ @taquito/rpc 13.0.1
  │     └─┬ @taquito/http-utils 13.0.1
  │       └── axios 0.26.1
  ├─┬ @taquito/rpc 13.0.1
  │ └─┬ @taquito/http-utils 13.0.1
  │   └── axios 0.26.1
  ├─┬ @taquito/taquito 13.0.1
  │ ├─┬ @taquito/http-utils 13.0.1
  │ │ └── axios 0.26.1
  │ ├─┬ @taquito/michelson-encoder 13.0.1
  │ │ └─┬ @taquito/rpc 13.0.1
  │ │   └─┬ @taquito/http-utils 13.0.1
  │ │     └── axios 0.26.1
  │ └─┬ @taquito/rpc 13.0.1
  │   └─┬ @taquito/http-utils 13.0.1
  │     └── axios 0.26.1
  ├── axios 1.6.7
  └─┬ tronweb 5.2.0
    └── axios 0.26.1
axios 1.6.7
```

**Comment by haammar-ledger** [comment]
https://github.com/LedgerHQ/ledger-live/pull/6299/commits/5acaee51294bf1226c260709db8d09e31c9dc1ff : tentative commit to update coin dependencies that have outdated axios version.

:warning: CHANGES ARE NOT TESTED YET, need careful non-reg :warning:

What this commit does:
- update tronweb
- update @cosmjs/*
- update @elrondnetwork/*
- update @taquito/*
- remove @keplr-wallet/*

Result of `pnpm desktop why axios` :
```
$ pnpm desktop why axios
dependencies:
@ledgerhq/coin-evm link:../../libs/coin-evm
├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
│ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│   └── axios 1.6.7
├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│ └── axios 1.6.7
├─┬ @ledgerhq/domain-service link:../../libs/domain-service
│ └── axios 1.6.7
├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
│ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
│ │ └── axios 1.6.7
│ └── axios 1.6.7
├─┬ @ledgerhq/live-network link:../../libs/live-network
│ └── axios 1.6.7
└── axios 1.6.7
@ledgerhq/coin-framework link:../../libs/coin-framework
└─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  └── axios 1.6.7
@ledgerhq/domain-service link:../../libs/domain-service
└── axios 1.6.7
@ledgerhq/hw-transport-http link:../../libs/ledgerjs/packages/hw-transport-http
└── axios 1.6.7

devDependencies:
@ledgerhq/test-utils link:../../libs/test-utils
└─┬ @ledgerhq/live-common link:../../libs/ledger-live-common
  ├─┬ @cosmjs/stargate 0.32.2
  │ └─┬ @cosmjs/tendermint-rpc 0.32.2
  │   └── axios 1.6.7
  ├─┬ @crypto-org-chain/chain-jslib 1.1.2
  │ ├─┬ @cosmjs/proto-signing 0.24.1
  │ │ └─┬ @cosmjs/launchpad 0.24.1
  │ │   └── axios 0.21.4
  │ ├─┬ @cosmjs/stargate 0.25.0-alpha.2
  │ │ └─┬ @cosmjs/tendermint-rpc 0.25.0-alpha.2
  │ │   └── axios 0.21.4
  │ ├─┬ @cosmjs/tendermint-rpc 0.25.0-alpha.2
  │ │ └── axios 0.21.4
  │ └── axios 0.21.2
  ├─┬ @elrondnetwork/erdjs-network-providers 1.2.0
  │ └── axios 0.24.0
  ├─┬ @ledgerhq/coin-algorand link:../../libs/coin-algorand
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-evm link:../../libs/coin-evm
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/domain-service link:../../libs/domain-service
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
  │ │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ │ └── axios 1.6.7
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/live-network link:../../libs/live-network
  │ │ └── axios 1.6.7
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-near link:../../libs/coin-near
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/coin-polkadot link:../../libs/coin-polkadot
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/hw-app-eth link:../../libs/ledgerjs/packages/hw-app-eth
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/domain-service link:../../libs/domain-service
  │ │ └── axios 1.6.7
  │ ├─┬ @ledgerhq/evm-tools link:../../libs/evm-tools
  │ │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ │ └── axios 1.6.7
  │ │ └── axios 1.6.7
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/hw-app-vet link:../../libs/ledgerjs/packages/hw-app-vet
  │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/hw-transport-node-speculos-http link:../../libs/ledgerjs/packages/hw-transport-node-speculos-http
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/live-countervalues link:../../libs/live-countervalues
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @ledgerhq/live-countervalues-react link:../../libs/live-countervalues-react
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-countervalues link:../../libs/live-countervalues
  │   ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │   │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   │   └── axios 1.6.7
  │   ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │   │ └── axios 1.6.7
  │   └─┬ @ledgerhq/live-network link:../../libs/live-network
  │     └── axios 1.6.7
  ├─┬ @ledgerhq/live-network link:../../libs/live-network
  │ └── axios 1.6.7
  ├─┬ @ledgerhq/live-nft link:../../libs/live-nft
  │ ├─┬ @ledgerhq/coin-framework link:../../libs/coin-framework
  │ │ └─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │   └── axios 1.6.7
  │ ├─┬ @ledgerhq/cryptoassets link:../../libs/ledgerjs/packages/cryptoassets
  │ │ └── axios 1.6.7
  │ └─┬ @ledgerhq/live-network link:../../libs/live-network
  │   └── axios 1.6.7
  ├─┬ @stellar/stellar-sdk 11.2.2
  │ └── axios 1.6.7
  ├── axios 1.6.7
  └─┬ tronweb 5.3.2
    └── axios 1.6.7
axios 1.6.7
```

Unfortunately, latest version still uses outdated axios for two dependencies :confused: :
- @crypto-org-chain/*
- @elrondnetwork/*

Need to contact these projects to ask them to update axios.

**Comment by gre** [comment]
with the latest commit, there is a lot of failure that arise, so i think we have more going on regarding the crypto libs update

**Pull request closed without merging**